Source/Core/DolphinQt/Debugger/MemoryWidget.cpp

```diff
@@ -611,3 +623,26 @@ void MemoryWidget::OnFindPreviousValue()
 {
   FindValue(false);
 }
+
+void MemoryWidget::OnFloatToHex(bool float_in)
+{
+  float float_val;
+  bool good;
+
+  if (float_in)
+  {
+    float_val = m_float_convert->text().toFloat(&good);
+    if (!good)
+      return;
+    u32* out = (u32*)&float_val;
```

Comment:
Please don't do this, use `memcpy` instead (or the handy `Common::BitCast` from `BitUtils.h`) to reinterpret data in one format as another.